src/legacy/core_plugins/kibana/server/tutorials/apm/envs/on_prem.js

```diff
@@ -155,14 +165,17 @@ export function onPremInstructions(apmIndexPattern) {
             defaultMessage: 'No data has been received from agents yet',
           }),
           esHitsCheck: {
-            index: apmIndexPattern,
+            index: [
+              config.get('apm_oss.errorIndices'),
+              config.get('apm_oss.transactionIndices'),
+              config.get('apm_oss.metricsIndices'),
+              config.get('apm_oss.sourcemapIndices'),
+            ],
```

Comment:
Changed this to actual indices. It was a legacy that we were using the index pattern title. We shouldn't be using this to query.